collect_app/src/androidTest/java/org/odk/collect/android/regression/FillBlankFormTest.java

```diff
@@ -52,6 +59,80 @@
             .around(new CopyFormRule("emptyGroupFieldList2.xml", "regression"))
             .around(new CopyFormRule("metadata2.xml", "regression"));
 
+    @Test
+    public void subtext_ShouldDisplayAdditionalInformation() {
+
+        //TestCase2
+        MainMenu.clickFillBlankForm();
+        MainMenu.checkIsFormSubtextDisplayed();
+
+    }
+
+    @Test
+    public void exitDialog_ShouldDisplaySaveAndIgnoreOptions() {
+
+        //TestCase6 , TestCase9
+        MainMenu.startBlankForm("All widgets");
+        pressBack();
+        FormEntry.checkIsStringDisplayed(R.string.keep_changes);
+        FormEntry.checkIsStringDisplayed(R.string.do_not_save);
+        FormEntry.clickOnString(R.string.do_not_save);
+        FormEntry.checkIsIdDisplayed(R.id.enter_data);
+        FormEntry.checkIsIdDisplayed(R.id.get_forms);
+
+    }
+
+    @Test
+    public void searchBar_ShouldSearchForm() {
+
+        //TestCase12
+        MainMenu.clickFillBlankForm();
+        MainMenu.clickMenuFilter();
+        MainMenu.searchInBar("Aaa");
+        pressBack();
+        pressBack();
+
+    }
+
+    @Test
+    public void navigationButtons_ShouldBeVisibleWhenAreSetInTheMiddleOfForm() {
```

Comment:
What do you think the policy should be for functionality that already has Espresso tests? See https://github.com/opendatakit/collect/blob/master/collect_app/src/androidTest/java/org/odk/collect/android/formentry/FormNavigationButtonTest.java for example. Do you think of these regression tests as entirely separate and so it's ok to have some redundancy between the two?